public/app/features/profile/ChangePasswordForm.tsx

```diff
@@ -120,3 +206,32 @@ export const ChangePasswordForm = ({ user, onChangePassword, isSaving }: Props)
     </div>
   );
 };
+
+export const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    label: css({
+      display: 'flex',
```

Comment:
```suggestion
      display: 'flex',
      alignItems: 'center',
```
This will align the icon with the text, as they are currently slightly misaligned.

Reply:
Changed as suggested!